indy-besu/vdr/src/client/client.rs

```diff
@@ -30,27 +33,40 @@ impl LedgerClient {
     ///  client to use for building and sending transactions
     pub fn new(
         chain_id: u64,
-        node_address: &str,
+        mut node_addresses: Vec<&str>,
```

Comment:
Should we better pass `quorum_config` to the client creation?

Reply:
Updated